src/language-css/utils/index.js

```diff
@@ -95,7 +95,7 @@ function insideAtRuleNode(path, atRuleNameOrAtRuleNames) {
 function insideURLFunctionInImportAtRuleNode(path) {
   const { node } = path;
   return (
-    node.groups[0].value === "url" &&
+    node.groups?.[0].value === "url" &&
```

Comment:
I don't know exactly which source code triggers the exception.
I encountered this condition when running prettier through Stylelint with their VSCode extension during or shortly after a switch between branches.

I mimicked the same code as found in `isKeyValuePairInParenGroupNode`.
Logically this change should prevent exceptions and should still work as intended.